### DISCUSSION TITLE
Select box is not showing outside of its container

### DISCUSSION THREAD

**Discussion by Tihitna-22** [mantinedev/mantine#4358] [Q&A] [answered]
This is the conponent i used to render parent object into the select box but i cant see data renderd out side of the container
     <Controller
        name="parentId"
        control={control}
        render={({ field }) => (
          <Select
            {...field}
            className="mb-2"
            label="Topic Parents "
            placeholder="Choose Parent"
            data={parent}
            maxDropdownHeight={280}
            disabled={props?.mode === 'archive' ? true : false}
            searchable
          />
        )}
      />

**Comment by rtivital** [comment, marked as answer]
Set `<Select withinPortal />`

**Reply by Tihitna-22** [reply]
Thanks for the help
